pkg/ebpf/c/tracee.bpf.c

```diff
@@ -2898,34 +2952,55 @@ int tracepoint__raw_syscalls__sys_exit(struct bpf_raw_tracepoint_args *ctx)
     if (sys->id != id)
         return 0;
 
+    sys->ret = ret;
+
+    bool submitExit = should_submit(RAW_SYS_EXIT, config);
+    bool submitSyscall = should_submit(sys->id, config);
+
+    if (submitExit || submitSyscall) {
+        bpf_tail_call(ctx, &sys_exit_submit_tail, 0);
+    }
```

Comment:
This should be removed - same reasons as in sys_enter

Reply:
see my comment [here](https://github.com/aquasecurity/tracee/pull/2080#discussion_r958348112) on that